### ISSUE TITLE
support-logging API json filed error

### ISSUE THREAD

**Issue by weichou1229** [edgexfoundry/edgex-go#372]
According to RML https://github.com/edgexfoundry/edgex-go/blob/master/support/logging/raml/support-logging.raml#L27
json:"loglevel" should be json:"logLevel" in
https://github.com/edgexfoundry/edgex-go/blob/master/support/domain/domain.go#L18

but now it's return
```
{
  "loglevel":"INFO",
  "labels":[
  ],
  "originService":"edgex-device-virtual",
  "message":"executing schedule 5b40baa69f8fc2000110f167 'interval-for-vr-9' at 2018-07-08T08:27:55Z[GMT]",
  "created":1531038475415
}
```

**Comment by cloudxxx8** [comment]
@weichou1229 this issue has been fixed by #402
please close this issue

**Comment by weichou1229** [comment]
close fixed issue
